tests/unit/test_routes.py

```diff
@@ -589,25 +589,6 @@ def test_validate_resource(module_client, module_db, fake_auth_from_oc):
 
     apikey = get_api_key(client)
 
-    # Input validation
-    # IntegrityError
-    response = client.put("/api/v1/resources/2",
```

Comment:
I don't know what this test does. It's a valid request as far as I can tell. In order to generate an IntegrityError, we would need to mock one.